website/docs/language/meta-arguments/lifecycle.mdx

```diff
@@ -144,6 +144,8 @@ The arguments available within a `lifecycle` block are `create_before_destroy`,
   }
   ```
 
+  If a local value, module output, or variable needs to be tracked by `replace_triggered_by`, it can be stored in a [`terraform_data`](language/resources/terraform-data) managed resource.
```

Comment:
I worry that some readers will not connect this statement with the rule above about requiring resource addresses and so won't understand the relevance of this paragraph. The following is my attempt to connect the two and also hint a bit about why this is true:

```suggestion
  `replace_triggered_by` allows only resource addresses because the decision is based on the planned actions for all of the given resources. Plain values such as local values or input variables do not have planned actions of their own, but you can treat them with a resource-like lifecycle by using them with [the `terraform_data` resource type](/language/resources/terraform-data).
```

This does feel kinda overly wordy so probably there's a compromise between what you wrote and what I wrote. :man_shrugging:

Reply:
That seems like as good an update as any right now. I'll let the writers take it on later.